test/onnx/test_pytorch_onnx_caffe2.py

```diff
@@ -77,6 +77,11 @@ def wrapper(self):
 def do_export(model, inputs, *args, **kwargs):
     f = io.BytesIO()
     out = torch.onnx._export(model, inputs, f, *args, **kwargs)
+    if isinstance(model, torch.jit.ScriptModule):
```

Comment:
Can we fix `out` (such as unpack) instead of directly rerun the model?

Reply:
I don't understand the "fix" in your comment... When exporting ScriptModule we don't run the model, we get `out=None`. The `out` here is used to compare against `caffe2_out`, so it shouldn't hurt running the model if it's ScriptModule?

Reply:
this part is not related to the "fix" in the PR title. It is basically updating the current test infra to be able to test ScriptModule models. The actual fix is in torch/csrc/jit/script/init.cpp _propagate_and_assign_input_and_output_shapes